Example/Controllers/SwiftUI/Features/Settings/Features/MiniAppSettingsContactsView.swift

```diff
@@ -98,6 +101,9 @@ extension MiniAppSettingsContactsView {
                     Text("Contact ID: \(contactId)")
                         .lineLimit(1)
                     Text("E-Mail address: \(email)")
+                    if !allEmails.isEmpty && allEmails.first != "" {
```

Comment:
```suggestion
                    if allEmails.first ?? "" != "" {
```

Reply:
Thank you :) I have update this in new commit.